lib/protocol/collection/item_collection.ts

```diff
@@ -117,11 +117,25 @@ export class ItemCollection extends MutableCollection<SNItem> {
       /** Filtered content type map */
       const filteredCTMap = this.filteredMap[contentType]!;
       const sortedElements = this.sortedMap[contentType]!;
+
+      const currentIndex = filteredCTMap[element.uuid];
+      const currentElement = !isNullOrUndefined(currentIndex) ? sortedElements[currentIndex] : undefined;
```

Comment:
I think `previous` would make more sense instead of current when reading the code. `previousIndex` and `previousElement`.